server/integration_tests/test_data/e2e_edge_cases/povertyincaliforniaandcalifornia/chart_config.json

```diff
@@ -659,67 +659,6 @@
             ],
             "denom": "Count_Person",
             "title": "Population With Poverty Status Determined"
-          },
```

Comment:
Count_Person_5OrMoreYears_OnlyEnglishSpokenAtHome_AbovePovertyLevelInThePast12Months got pushed out of the list of matches because sdg topics about poverty got added